### PR TITLE
External db instructions

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

File: ncvoter/local_settings.example.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+DATABASES['default'].update({
+    'PORT': 5455,
+    'NAME': 'ncvotes',
+    'USER': '',
+    'PASSWORD': '',
+})
+
+NCVOTER_DOWNLOAD_PATH = "/Volumes/Untitled/downloads/ncvoter"
+NCVHIS_DOWNLOAD_PATH = "/Volumes/Untitled/downloads/ncvhis"
```
Comment (vkurup) [inline ncvoter/local_settings.example.py]: We use 'Untitled' here and in the docs, but 'CalvinNCVdb' in the shell script. Should we stick to one name throughout, to make it clearer to the developer?

File: README.md [hunk 2]
```diff
@@ -29,6 +29,54 @@
...
+#### Creating an external database
+
+You can move only one database on your postgres instance to a different drive,
+but you can't stop and start single databases. This means simply moving the TABLESPACE
+would still leave you with problems when disconnecting the external drive.
+
+Instead, create a whole separate cluster and run a separate postgres instance *only*
+when working on this project.
+
+    pg_ctl -D /Volumes/Untitled/postgres initdb
```
Comment (vkurup) [inline README.md]: so `pg_ctl` is not on the PATH by default for Ubuntu, [by design](https://askubuntu.com/a/385970/7131) (sigh)

Could we just add a note, something like, 'For Ubuntu, all of the postgresql binaries are located in /usr/lib/postgresql/9.6/bin, so you'll need to type out the full path or add that directory to your PATH environment variable'?

File: voter/management/commands/voter_fetch_snapshot.py [hunk 3]
```diff
@@ -14,10 +17,6 @@
 s3client = boto3.client('s3')
 s3client.meta.events.register('choose-signer.s3.*', botocore.handlers.disable_signing)
 
-
-NCVOTER_ZIP_URL_BASE = "https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/"
```
Comment (vkurup) [inline voter/management/commands/voter_fetch_snapshot.py]: This URL is different than the one in voter_fetch.py.

* voter_fetch: http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip
* voter_fetch_snapshot: https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/

At a minimum, we need to remove the trailing `/ncvoter` from NCVOTER_ZIP_URL_BASE and add `/Snapshots/`

Comment (vkurup) [reply voter/management/commands/voter_fetch_snapshot.py]: The beginning part of the URL is also different, but maybe that part doesn't matter (they probably point to the same thing?)

Comment (rtburg) [reply voter/management/commands/voter_fetch_snapshot.py]: Those two URLs point to different things. The "historic" snapshots are in https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/. The live (aka current) voter registration file is at http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip

Comment (vkurup) [reply voter/management/commands/voter_fetch_snapshot.py]: @rtburg yes! my second comment refers to the fact that http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip and https://s3.amazonaws.com/dl.ncsbe.gov/data/ncvoter_Statewide.zip are the same file (even though the beginning of the URL looks different).

Comment (rtburg) [reply voter/management/commands/voter_fetch_snapshot.py]: Roger that. Carry on.

**Pull request merged**